helios.js

```diff
@@ -595,7 +595,7 @@ function stringToBytes(str) {
  * @param {number[]} bytes 
  * @returns {string}
  */
-function bytesToString(bytes) {
+export function bytesToString(bytes) {
```

Comment:
I liked that you named the function in PicoSwap `hexToText` originally. Do you think we should name this `bytesToText` ? ('text' seems less ambiguous than 'string')

Reply:
Sure, I like that idea 👍